gr-pdu/python/pdu/qa_time_delta.py

```diff
@@ -88,8 +88,8 @@ def test_002_normal(self):
         time_tag = pmt.dict_ref(a_meta, pmt.intern("system_time"), pmt.PMT_NIL)
         delta_tag = pmt.dict_ref(a_meta, pmt.intern(
             "sys time delta (ms)"), pmt.PMT_NIL)
-        self.assertAlmostEqual(tnow, pmt.to_double(time_tag), delta=60)
-        self.assertAlmostEqual(10000, pmt.to_double(delta_tag), delta=10)
+        self.assertAlmostEqual(tnow - 10.0, pmt.to_double(time_tag), delta=1e-6)
+        self.assertAlmostEqual(10000, pmt.to_double(delta_tag), delta=50)
```

Comment:
Recent test failures have a delta of less than 20 ms, so hopefully 50 will be enough to prevent failures.